src/filters/__test__/freaksFilter.test.jsx

```diff
@@ -11,10 +11,7 @@ const catFilters = (f1, f2) => ({
   projects: [ ...f1.projects, ...f2.projects ],
 });
 
-const getNamesOfFilteredFreaks = (freaks, filters) => getFilteredFreaks(freaks, filters).map((freak) => [
-  freak.firstName,
-  freak.lastName,
-]);
+const getNamesOfFilteredFreaks = (freaks, filters) => getFilteredFreaks(freaks, filters).map((freak) => [ freak.firstName, freak.lastName ]);
```

Comment:
I think it would be simpler for you to separate these functions into separate concerns
- one you already have, it filters
- the other extracts the name into a tuple

The second function could just be

```js
const makeFreaksNames = (freaks) => freaks.map((freak) => [ freak.firstName, freak.lastName ]);
```

further you could do a function that makes a name tuple:

```js
const makeNameTuple = (freak) => ([ freak.firstName, freak.lastName ]);
const makeFreaksNames = (freaks) => freaks.map(makeNameTuple);
```

then you would do
```js
makeNameTuples(getFilteredFreaks(freaks, filters))
```